C/tests/c4KeyTest.cc

```diff
@@ -46,22 +46,22 @@ class C4KeyTest : public CppUnit::TestFixture {
     void testReadKey() {
         populateKey();
         C4KeyReader r = c4key_read(key);
-        AssertEqual(c4key_peek(&r), kC4Array);
+        AssertEqual(c4key_peek(&r), (uint8_t)kC4Array);
```

Comment:
These changes cause compile errors in Clang :( because `C4KeyToken` is not the same type as `uint8_t`.